lib/preload.js

```diff
@@ -427,6 +428,7 @@ class Preloader extends EventEmitter {
 				this.freeSpace = info.free_space;
 				this.preloadedBuilds = info.preloaded_builds;
 				this.supervisorVersion = info.supervisor_version;
+				this.balenaOSVersion = info.balenaOS_version;
```

Comment:
This version is extracted from the `/etc/issue` file in the preloaded balenaOS image.